android/dependencies.gradle

```diff
@@ -1,9 +1,9 @@
 ext {
     apply(from: "${buildscript.sourceFile.getParent()}/test-app-util.gradle")
 
-    compileSdkVersion = 34
-    minSdkVersion = 23
-    targetSdkVersion = 33
+    compileSdkVersion = rootProject.findProperty("react.compileSdkVersion") ?: 34
+    minSdkVersion = rootProject.findProperty("react.minSdkVersion") ?: 23
+    targetSdkVersion = rootProject.findProperty("react.targetSdkVersion") ?: 33
```

Comment:
Nit: Save the defaults as constants? Unless you can't.

Reply:
This kinda is the file that is supposed to have all the defaults 😄